nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ControlRate.java

```diff
@@ -228,12 +238,13 @@ public void onScheduled(final ProcessContext context) {
         rateControlAttribute = context.getProperty(RATE_CONTROL_ATTRIBUTE_NAME).getValue();
         maximumRateStr = context.getProperty(MAX_RATE).getValue().toUpperCase();
         groupingAttributeName = context.getProperty(GROUPING_ATTRIBUTE_NAME).getValue();
+        maxFlowFilePerTrigger = context.getProperty(MAX_FF_PER_TRIGGER).getValue();
```

Comment:
This should probably be defined as an int, rather than a String, and can then just use context.getProperty().asInteger(). But I really prefer to remove this property all together.